samples/advanced-markers-basic-style/index.ts

```diff
@@ -6,27 +6,22 @@
 
 // [START maps_advanced_markers_basic_style]
 const parser = new DOMParser();
+const mapElement = document.querySelector('gmp-map') as google.maps.MapElement;
 
 async function initMap() {
     // Request needed libraries.
     const { Map } = await google.maps.importLibrary("maps") as google.maps.MapsLibrary;
     const { AdvancedMarkerElement, PinElement } = await google.maps.importLibrary("marker") as google.maps.MarkerLibrary;
 
-    const map = new Map(document.getElementById('map') as HTMLElement, {
-        center: { lat: 37.419, lng: -122.02 },
-        zoom: 14,
-        mapId: '4504f8b37365c3d0',
-    });
-
     // Each PinElement is paired with a MarkerView to demonstrate setting each parameter.
 
     // [START maps_advanced_markers_basic_style_title]
     // Default marker with title text (no PinElement).
     const markerViewWithText = new AdvancedMarkerElement({
```

Comment:
the term "markerView" comes from the original api design, but since we are using web component, we can drop the "view", could you replace all the "markerView" to "marker"?